contracts/libraries/HyperdriveExecution.sol

```diff
@@ -0,0 +1,578 @@
+            effectiveShareReserves,
+            bondReserves,
+            poolConfig.initialVaultSharePrice,
+            poolConfig.timeStretch
+        );
+        (, bondReservesDelta, ) = _calculateOpenLongFees(
+            _shareAmount,
+            bondReservesDelta,
+            _vaultSharePrice,
+            spotPrice,
+            poolConfig.fees.curve,
+            poolConfig.fees.governanceLP
+        );
+        return bondReservesDelta;
+    }
+
+    /// @dev Calculate the fees involved with opening the long and apply them.
+    /// @param _shareReservesDelta The change in the share reserves without fees.
+    /// @param _bondReservesDelta The change in the bond reserves without fees.
+    /// @param _vaultSharePrice The current vault share price.
+    /// @param _spotPrice The current spot price.
+    /// @return The change in the share reserves with fees.
+    /// @return The change in the bond reserves with fees.
+    /// @return The governance fee in shares.
+    function _calculateOpenLongFees(
+        uint256 _shareReservesDelta,
+        uint256 _bondReservesDelta,
+        uint256 _vaultSharePrice,
+        uint256 _spotPrice,
+        uint256 _curveFee,
+        uint256 _governanceLPFee
+    ) internal pure returns (uint256, uint256, uint256) {
+        // Calculate the fees charged to the user (curveFee) and the portion
+        // of those fees that are paid to governance (governanceCurveFee).
+        (
+            uint256 curveFee, // bonds
+            uint256 governanceCurveFee // bonds
+        ) = _calculateFeesGivenShares(
+                _shareReservesDelta,
+                _spotPrice,
+                _vaultSharePrice,
+                _curveFee,
+                _governanceLPFee
+            );
+
+        // Calculate the impact of the curve fee on the bond reserves. The curve
+        // fee benefits the LPs by causing less bonds to be deducted from the
+        // bond reserves.
+        _bondReservesDelta -= curveFee;
+
+        // NOTE: Round down to underestimate the governance fee.
+        //
+        // Calculate the fees owed to governance in shares. Open longs are
+        // calculated entirely on the curve so the curve fee is the total
+        // governance fee. In order to convert it to shares we need to multiply
+        // it by the spot price and divide it by the vault share price:
+        //
+        // shares = (bonds * base/bonds) / (base/shares)
+        // shares = bonds * shares/bonds
+        // shares = shares
+        uint256 totalGovernanceFee = governanceCurveFee.mulDivDown(
+            _spotPrice,
+            _vaultSharePrice
+        );
+
+        // Calculate the number of shares to add to the shareReserves.
+        // shareReservesDelta, _shareAmount and totalGovernanceFee
+        // are all denominated in shares:
+        //
+        // shares = shares - shares
+        _shareReservesDelta -= totalGovernanceFee;
```

Comment:
I could be missing where these are used, but if we're not going to use these, we might as well not calculate them. I get that it might be helpful to keep this around for future PRs, but adding a TODO as a reminder would be a good idea.